astro/src/content/quickstarts/quickstart-javascript-express-web.mdx

```diff
@@ -0,0 +1,259 @@
+
+In that case, the system might look like this before FusionAuth is introduced.
+
+<LoginBefore alt={"Request flow during login before FusionAuth"}/>
+
+The login flow will look like this after FusionAuth is introduced.
+
+<LoginAfter alt={"Request flow during login after FusionAuth"}/>
+
+In general, you are introducing FusionAuth in order to normalize and consolidate user data. This helps make sure it is consistent and up-to-date as well as offloading your login security and functionality to FusionAuth.
+
+## Getting Started
+In this section, you'll get FusionAuth up and running, and configured with the ChangeBank application.
+
+### Clone the Code
+First off, grab the code from the repository and change into that directory.
+
+```
+git clone https://github.com/FusionAuth/fusionauth-quickstart-javascript-express-web.git
+cd fusionauth-quickstart-javascript-express-web
+```
+
+### Run FusionAuth via Docker
+
+In the root directory of the repo you'll find a Docker compose file (docker-compose.yml) and an environment
+variables configuration file (.env). Assuming you have Docker installed on your machine, you can stand up
+FusionAuth up on your machine with:
+
+```
+docker compose up -d
+```
+
+This will start three containers, once each for FusionAuth, Postgres, and Elastic.
+
+Here you are using a bootstrapping feature of FusionAuth, called [Kickstart](/docs/v1/tech/installation-guide/kickstart).
+When FusionAuth comes up for the first time, it will look at the `kickstart/kickstart.json` file and
+configure FusionAuth to a certain initial state.
+
+<Aside type="note">
+If you ever want to reset the FusionAuth system, delete the volumes created by Docker Compose by
+executing `docker compose down -v`, then re-run `docker compose up -d`.
+</Aside>
+
+FusionAuth will be configured with these settings:
+* Your client Id is `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`.
+* Your client secret is `2HYT86lWSAntc-mvtHLX5XXEpk9ThcqZb4YEh65CLjA-not-for-prod`.
+* Your example username is `richard@example.com` and the password is `password`.
+* Your admin username is `admin@example.com` and the password is `password`.
+* The base URL of FusionAuth `http://localhost:9011/`.
+
+You can log into the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want, but with Docker/Kickstart you don’t need to.
+
+### Setup Application
+
+In this section, you'll set up a basic Express application with a single page.
+
+Create a new directory to hold your application, and go into it.
+
+```shell
+mkdir changebank
+cd changebank
+```
+
+Create a file called `package.json` with the below dependencies and scripts. Feel free to update the `author` or other attributes as needed. 
+
+The `@fusionauth/typescript-client` will be used to exchange the OAuth Code for Access Token using PKCE, and also to retrieve the User using JWT.
+
+`package.json`
```

Comment:
I'd rather not have the filename here like this. That is inconsistent with other quickstarts.

Reply:
Or we can change it across all of them, but that's probably a clean up task.